lib/tasks/bulk.rake

```diff
@@ -0,0 +1,13 @@
+namespace :bulk do
+  desc "Send a bulk email to many subscriber lists"
+  task :email, %i(subject body) => :environment do |_t, args|
+    subscriber_list_ids = args.extras
```

Comment:
Have you experimented with how this might get populated? I wonder if we're planning to do pre-processing on this or plan to have something through that we process into the list?